test/util/bitcoin-util-test.py

```diff
@@ -48,7 +48,7 @@ def main():
 
 def bctester(testDir, input_basename, buildenv):
     """ Loads and parses the input file, runs all tests and reports results"""
-    input_filename = testDir + "/" + input_basename
+    input_filename = os.path.join(testDir, input_basename)
```

Comment:
Might as well pass in `input_filename` to `bctester` and instead do the path.join in line 47, no?